packages/bruno-app/src/components/Devtools/Console/TerminalTab/SessionList.js

```
@@ -0,0 +1,151 @@
+
+  return (
+    <StyledSessionList>
+      {sessions.map((session) => {
+        const { name } = getSessionDisplayInfo(session);
+        return (
+          <ToolHint
+            text={getFullPath(session)}
+            toolhintId={`session-path-${session.sessionId}`}
+            place="bottom-start"
+            delayShow={100}
+          >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)